group/nist/qrsuite.go

```diff
@@ -26,9 +26,8 @@ func (s QrSuite) Hash() hash.Hash {
 	return sha256.New()
 }
 
-// SHA3/SHAKE128 Sponge Cipher
-func (s QrSuite) Cipher(key []byte, options ...interface{}) kyber.Cipher {
-	return sha3.NewShakeCipher128(key, options...)
+func (s QrSuite) XOF(key []byte) kyber.XOF {
```

Comment:
Same thing as in ed25519 - perhaps change the name of the suite to:

`NewBlakeSHA256QR512`
?

Reply:
Fixed.